src/components/pages/People.vue

```diff
@@ -203,6 +209,17 @@ export default {
       } else {
         return ''
       }
+    },
+
+    filteredPeople () {
+      const persons = []
+      this.displayedPeople.forEach(people => {
+        let person = ''
+        person += people.first_name
+        person += people.last_name
+        persons.push(person)
```

Comment:
You should use a map. Relying on `includes` is too slow. It leads to a O(n2) complexity.

```
const personKey = `${people.first_name}-${people.last_name}`
persons[personKey] = true
```